database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/source/DBSource.java

```diff
@@ -136,20 +136,24 @@ private String getJDBCPluginType() {
    */
   @Path("getSchema")
```

Comment:
ideally, but we don't have option to specify the httpMethod here. as all requests are POST currently, as we pass parameter for the method in Http request body.